plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/persistence/entity/CouponAndProductCategoryEntity.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.fluxc.persistence.entity
+
+import androidx.room.Entity
+import androidx.room.ForeignKey
+import androidx.room.Index
+
+@Entity(
+    tableName = "CouponsAndProductCategories",
+    foreignKeys = [
+        ForeignKey(
+                entity = CouponEntity::class,
+                parentColumns = arrayOf("id", "siteId"),
+                childColumns = arrayOf("couponId", "siteId"),
```

Comment:
⛏️ Minor, but you can use also a shorter, to me more readable form 
```suggestion
                parentColumns = ["id", "siteId"],
                childColumns = ["couponId", "siteId"],
```